code/modules/clothing/under/marine_uniform.dm

```diff
@@ -46,6 +51,7 @@
 	adjustment_variants = list()
 	has_sensor = 2
 	flags_item_map_variant = null
+	adjustment_variants = list()
```

Comment:
this is overwriting the `adjustment_variants` right above it.
look two lines up.